vcpkg/TOC.yml

```
@@ -370,6 +370,8 @@
           href: maintainers/functions/internal/z_vcpkg_setup_pkgconfig_path.md
     - name: Portfile variables
       href: maintainers/variables.md
+    - name: Software bill of materials generation

```

```suggestion
    - name: Software Bill of Materials generation
```
I believe "Software Bill of Materials" is supposed to be title case, so we should use this consistently.